pkgs/games/otto-matic/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "OttoMatic";
-  version = "4.0.1";
+  version = "unstable-2023-11-13";
```

Comment:
maybe we should change updater  script, otherwise @r-ryantm will downgrade it back to 4.0.1

also technically version should be `4.0.1-unstable-2023-11-13`
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming